Bottom: e3ba019abfcdc9c9fcc9804b920eeb7f5dfb106e
Top:    ffeb3ebdc86ded3a8082851b630e5bb0af026d0f
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 13:40:29 -0500

Refresh of exim4-install-key-crt

---
diff --git a/docker/docker-exim4/docker-entrypoint.sh b/docker/docker-exim4/docker-entrypoint.sh
index 3c3e0a26..77a575ee 100755
--- a/docker/docker-exim4/docker-entrypoint.sh
+++ b/docker/docker-exim4/docker-entrypoint.sh
@@ -25,17 +25,11 @@ case ${1} in
 
         # copy certificate
         [ -e /etc/ssl/private/exim.crt ] \
-            && rm -f ${CONFIG_PATH}/exim.crt \
-            && cp $(realpath /etc/ssl/private/exim.crt) ${CONFIG_PATH}/exim.crt \
-            && chmod 640 ${CONFIG_PATH}/exim.crt \
-            && chown root:Debian-exim ${CONFIG_PATH}/exim.crt
+            && install -oroot -gDebian-exim -m640 $(realpath /etc/ssl/private/exim.crt) ${CONFIG_PATH}/exim.crt
 
         # copy key
         [ -e /etc/ssl/private/exim.key ] \
-            && rm -f ${CONFIG_PATH}/exim.key \
-            && cp $(realpath /etc/ssl/private/exim.key) ${CONFIG_PATH}/exim.key \
-            && chmod 640 ${CONFIG_PATH}/exim.key \
-            && chown root:Debian-exim ${CONFIG_PATH}/exim.key
+            && install -oroot -gDebian-exim -m640 $(realpath /etc/ssl/private/exim.key) ${CONFIG_PATH}/exim.key
 
         # run the debian utility to update/generate the configuration
         $(which update-exim4.conf) --verbose
